gui/slick/views/addShows_popularShows.mako

```diff
@@ -35,25 +35,19 @@
 <br>
 <div id="popularShows">
     <div id="container">
-    % if not popular_shows:
+% if not popular_shows:
```

Comment:
@medariox i would keep the space here as this code belongs to the `<div id="container">`